### PR TITLE
feat: added rule no-tabindex-positive

### PR DESCRIPTION
**Pull request by ematipico**
As per title, this PR implements the upstream [rule](https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/tabindex-no-positive.md).

Refs to #341

### CODE REVIEW

File: packages/@romejs/diagnostics/categories.ts [hunk 1]
```diff
@@ -151,6 +151,7 @@ type LintDiagnosticCategory =
 	| "lint/jsx-a11y/noAccessKey"
 	| "lint/jsx-a11y/noAutofocus"
 	| "lint/jsx-a11y/noDistractingElements"
+	| "lint/jsx-a11y/noTabindexPositive"
```
Comment (bitpshr) [inline packages/@romejs/diagnostics/categories.ts:154]: Nit: Could this rule's name (and filenames) be updated to match the upstream rule? E.g. `jsx-a11y/tabindexNoPositive` vs. `jsx-a11y/noTabindexPositive`.

File: packages/@romejs/diagnostics/descriptions.ts [hunk 2]
```diff
@@ -292,6 +292,10 @@ export const descriptions = createMessages({
 	},
 	// @romejs/js-compiler
 	LINT: {
+		JSX_A11Y_TABINDEX_NO_POSITIVE: {
+			category: "lint/jsx-a11y/noTabindexPositive",
+			message: "No positive tab index",
```
Comment (bitpshr) [inline packages/@romejs/diagnostics/descriptions.ts:297]:
```suggestion
			message: "Avoid positive integer values for <emphasis>tabIndex</emphasis>.",
```

File: packages/@romejs/diagnostics/categories.ts (hunk 1 above)
Comment (ematipico) [reply packages/@romejs/diagnostics/categories.ts:154]: Ahaha! I had created that at the beginning 🤣 
No biggie, I can revert the changes!

**Comment by sebmck** [comment]
Sweet! Thank you!

**Pull request merged**